client/src/components/Footer/style.js

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const Container = styled('div')`
+  padding: ${({ theme }) => `${theme.spacings[3]} ${theme.spacings[5]}`};
```

Comment:
the padding on the desktop doesn't match the design
```suggestion
export const Container = styled('div')`
    padding: ${({ theme: { spacings, constants } }) =>
    `${spacings[3]} ${constants.general.paddingLeft}`};
  ${({ theme }) => theme.media.mobile} {
    padding: ${({ theme }) => `${theme.spacings[3]} ${theme.spacings[5]}`};
  }
  
```